pkg/operator/ceph/file/mds/livenessprobe.sh

```diff
@@ -6,7 +6,7 @@ MDS_ID="{{ .MdsId }}"
 FILESYSTEM_NAME="{{ .FilesystemName }}"
 KEYRING="{{ .Keyring }}"
 
-outp="$(ceph fs dump --mon-host="$ROOK_CEPH_MON_HOST" --mon-initial-members="$ROOK_CEPH_MON_INITIAL_MEMBERS" --keyring "$KEYRING" --format json)"
+outp="$(ceph fs dump --mon-host="$ROOK_CEPH_MON_HOST" --mon-initial-members="$ROOK_CEPH_MON_INITIAL_MEMBERS" --keyring "$KEYRING" --format json --connect-timeout 15)"
```

Comment:
we really need to have a discussion on this, generally we do 15s 

Looking at other liveness probes value for mds
```
		// timeout for the mds cmd, should be less than mdsTimeoutSeconds
	mdsCmdTimeout int32 = 20
```
So the wait to connect should be less than that, so I believe any 15,10 should be the right value, but I will let @BlaineEXE to also take a look